HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServices.json

```diff
@@ -0,0 +1,238 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "HomeHealthServices",
+  "name": "HomeHealthServices",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/cqf-questionnaire",
+      "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-r4"
+    ]
+  },
+  "title": "Home Health Services",
+  "status": "draft",
+  "subjectType": [
+    "Patient"
+  ],
+  "date": "2019-11-14",
+  "publisher": "Da Vinci DTR",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/HomeHealthServices-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+    }
+  ],
+  "contained": [
+    {
+      "resourceType": "ValueSet",
+      "id": "gender",
+      "url": "#gender",
+      "status": "draft",
+      "expansion": {
+        "contains": [
+          {
+            "code": "male",
+            "display": "Male"
+          },
+          {
+            "code": "female",
+            "display": "Female"
+          },
+          {
+            "code": "other",
+            "display": "Other"
+          }
+        ]
+      }
+    }
+  ],
+  "item": [
+    {
+      "linkId": "1",
```

Comment:
Use PatientInfo questionnaire

Reply:
@yunwwang This questionnaire is WIP and Howard is still working on it. My PR was originally created based on his branch and sort diverged from there. So please ignore anything in Questionnaire-R4-HomeHealthServices.json and HomeHealthServicesPrepopulation-0.0.1.cql. I should have indicated in the PR description. Sorry about that.